Lib/fontTools/ttLib/ttFont.py

```diff
@@ -138,9 +151,13 @@ def close(self):
 			self.reader.close()
 
 	def save(self, file, reorderTables=True):
-		"""Save the font to disk. Similarly to the constructor,
-		the 'file' argument can be either a pathname or a writable
-		file object.
+		"""Save the font to disk.
+
+		Args:
+			file: Similarly to the constructor, can be either a pathname or a writable
+				file object.
+			reorderTables (bool): If true (the default), reorder the tables, sorting
```

Comment:
reorderTables is an Optional[bool]. When it's set to None (e.g. the subsetter does that), it means write tables in whatever order is most convenient (mostly due to inter-table dependencies at compile time) without caring about either the canonical order or the original font order. This should be the fastest mode